src/main/resources/messages.yml

```diff
@@ -6,6 +6,7 @@
 #   - CHAT: Chat
 #   - ACTION_BAR: Action Bar
 #   - SUBTITLE: Subtitle
+#   - added this
```

Comment:
This line isnt needed.

Reply:
well i know i need it for to push it as new commit on discord, you need to change it later

Reply:
well i know but i need that to push it.....,

Reply:
change it your self i suppose.

Reply:
> well i know but i need that to push it.....,

you still need to fix the other placeholders, `%ah_count%` and `%ah_countdown%`. @abhiram555

Reply:
???

Reply:
i need to remove it from messages or what?

Reply:
> 
> 
> i need to remove it from messages or what?

you said that you [cant implement them](https://github.com/squirrelnetmc/AdvancementHunt/pull/18#discussion_r551033506), so change them back to what they were before